week-1/L-functions-nested/exercise2.js

```diff
@@ -1,5 +1,19 @@
+function greetings() {
+  return "Hello";
+}
+
+function upperCaseNames(name) {
+  return name.toUpperCase();
+}
+
 var mentor1 = "Daniel";
 var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+console.log(greetings() + " " + upperCaseNames(mentor1));
+console.log(greetings() + " " + upperCaseNames(mentor2));
+console.log(greetings() + " " + upperCaseNames(mentor3));
+console.log(greetings() + " " + upperCaseNames(mentor4));
+console.log(greetings() + " " + upperCaseNames(mentor5));
```

Comment:
You could avoid the empty string by leaving a space after `"Hello "` or before the name of the mentor`" Daniel"`. The expected outcome was `"HELLO DANIEL`..., your programme only changes the name to uppercase, change it to do that to the greeting and name